pkg/codegen/python/doc.go

```diff
@@ -52,6 +52,13 @@ func (d DocLanguageHelper) GetDocLinkForFunctionInputOrOutputType(pkg *schema.Pa
 	return ""
 }
 
+// GetDocLinkForBuiltInType returns the Python URL for a built-in type.
+// Currently not using the typeName parameter because the returned link takes to a general
+// top-level page containing info for all built in types.
+func (d DocLanguageHelper) GetDocLinkForBuiltInType(typeName string) string {
+	return "https://docs.python.org/3/library/stdtypes.html"
```

Comment:
@sean1588, why can't we be more specific? For example, for string, I'd expect to link directly to https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str. For numeric types, link directly to https://docs.python.org/3/library/stdtypes.html#numeric-types-int-float-complex. Etc.